src/bento/profileData.js

```diff
@@ -20,6 +20,26 @@ const editTool = {
   alt: 'edit',
 };
 
+const ignoredArms = ['revoked', 'rejected'];
+
+/**
+ * Gets the display format for the authenticator.
+ * @param {string} loginNm - The IDP used to login.
+ * */
+export const getAuthenticatorName = (loginNm) => {
```

Comment:
@ogwuazors Similarly like #610 can we  use this function from utils folder?

Reply:
@iksheth Should be resolved now.